smallcloudai/refact-chat-js
# tool images

# Tool images


## Description

when using the `chrome` tool, tools can respond with and array of `{m_type: "image/...", m_content: "base64string:"}`

## Type of change




## How to Test

See https://refact.fibery.io/Software_Development/UI-UX-133#Task/Images-coming-from-tools-207 for set up istructions.


1. Ask the chat to take a screen shot
2. It should respond with the image in a tool call

Note: at the moment the lsp transforms the result to a user message

## Screenshots (if applicable)

<img width="765" alt="Screenshot 2024-10-29 at 16 05 24" src="https://github.com/user-attachments/assets/d6156898-98af-4cfc-8a69-027d019b4c66">

## Checklist




## Linked Issues
https://refact.fibery.io/Software_Development/UI-UX-133#Task/Images-coming-from-tools-207
## Additional Notes

This may change as the lsp updated.
